tic_tac_toe/game.py

```diff
@@ -33,7 +43,11 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    for outer_list in board:
+        for inner_list in outer_list:
+            if inner_list == "-":
+                return False
```

Comment:
I think it is a good code. Maybe the variable names are a little confusing? Instead of outer and inner you could have user "row" and "column".

Also, I see you have use list comprehension on line 34, why didn't you use it over here? (Note that this is not a problem at all. I'm just wondering?)